test/integration/tasks/fleet_validate_test.go

```diff
@@ -287,7 +275,7 @@ var _ = Describe("FleetValidate", func() {
 			Expect(*fleet.Status.Conditions).To(HaveLen(1))
 			Expect((*fleet.Status.Conditions)[0].Type).To(Equal(api.FleetValid))
 			Expect((*fleet.Status.Conditions)[0].Status).To(Equal(api.ConditionStatusFalse))
-			Expect(*((*fleet.Status.Conditions)[0].Message)).To(Equal("fleet has 1 invalid configuration: <unknown>"))
+			Expect(*((*fleet.Status.Conditions)[0].Message)).To(Equal("1 invalid configuration: <unknown>. Error: failed to find configuration item name: unsupported discriminator: InvalidProviderSpec"))
```

Comment:
FYI @celdrake , this is the improvement we discussed

Reply:
Thanks, I checked and I could see the message when for example passing an invalid ignition version.